clang/test/AST/ast-dump-expr.cpp

```diff
@@ -583,3 +583,8 @@ void NonADLCall3() {
   f(x);
 }
 } // namespace test_adl_call_three
+
```

Comment:
nit: wrap the code within `namespace GH35300 { ...}`.

Reply:
Done